secboot/secboot_tpm.go

```diff
@@ -76,6 +79,11 @@ func isTPMEnabledImpl(tpm *sb.TPMConnection) bool {
 	return tpm.IsEnabled()
 }
 
```

Comment:
the changes to this file and tests are the main things I should preserve and rework in this PR, splitting them between here and kernel/fde.